src/config/config.go

```diff
@@ -40,6 +42,7 @@ func ProcessArgs() {
 	flag.BoolVar(&ret.Verbose, "verbose", false, "Increase verbosity, set log level to debug")
 	flag.StringVar(&ret.ControllerImage, "controller-image", "quay.io/ocpmetal/assisted-installer-controller:latest",
 		"Assisted Installer Controller image URL")
+	flag.DurationVar(&ret.InstallationTimeout, "installation-timeout", 120, "Installation timeout in minutes")
```

Comment:
```suggestion
	flag.DurationVar(&ret.InstallationTimeout, "installation-timeout", 120 * time.Minutes, "Installation timeout in minutes")
```

Reply:
otherwise it will be miliseconds

Reply:
https://github.com/eranco74/assisted-installer/pull/89/files#diff-8bea30c4cde4eea8efce766b4de08e43R281
I want to user to specify minutes.

Reply:
then it's probably should be IntVal it will be clearer